packages/System/src/index.tsx

```diff
@@ -57,11 +56,11 @@ const WRAPPER_PROPS = Object.freeze([
 ])
 
 /**
- * @deprecated use system from @xstyled/syled-components instead
+ * @deprecated use system from @wttj/xstyled-syled-components instead
  */
 export const system = compose<WuiSystemProps>(...SYSTEM_PROPS)
 /**
```

Comment:
since we've stopped using it in favor of *Box components; should we remove it from the exports ?

Reply:
We still need it to be able to override style props in particular use cases